### PR TITLE
working with pipelines update

### PR DESCRIPTION
**Pull request by t-koman**
For #373 ID editing and update concerning TSL verification for image registry access in pipelines

### CODE REVIEW

File: publish/working-with-stacks/working-with-stacks.md [hunk 1]
```diff
@@ -308,3 +308,74 @@ Follow these steps:
     ```
 
 Applications that have been developed using your stack can now be deployed to your Kubernetes cluster.
+
+## Transport layer security (TLS) verification for image registry access in pipelines
+
+when accessing image registries to pull and push images, pipelines enable the configuration creation of the cluster resource, [image.config.openshift.io/cluster](https://docs.openshift.com/container-platform/4.2/openshift_images/image-configuration.html#images-configuration-file_image-configuration).  Scenarios are presented of image registries that can be used and the setup steps required.
```
Comment (jantley-ibm) [inline publish/working-with-stacks/working-with-stacks.md:314]: Capitalize first word ("when")

File: publish/working-with-stacks/working-with-stacks.md [hunk 2]
```diff
@@ -308,3 +308,74 @@ Follow these steps:
...
+
+### Enable TLS verification for image registry access:
+
+#### Private image registries:
+ 
+If you use a private image registry and your registry uses certificates that are signed by trusted CA authorities, no further configuration is needed to enable TLS verification. Review the default truststore on the nodes of your cluster to ensure that you have the CA of your certificate in the list. With a self-signed certificates or a certificate that is signed by a CA, you must ensure that either certificate is listed in the default truststore. Steps are presented to add the CA to the trusted list so that when you run the pipelines, these CA certificates in the list are used.
+
+ - Ensure that you have access to the ca.crt files for your private registries.
```
Comment (jantley-ibm) [inline publish/working-with-stacks/working-with-stacks.md:322]: change
ca.crt
to
`ca.crt`
(using single angle quotes, like in next sentence)

**Review by jantley-ibm — COMMENTED** [review]
@t-koman just a couple of minor changes for me, thanks

File: publish/working-with-stacks/working-with-stacks.md (hunk 2 above)
Comment (t-koman) [reply publish/working-with-stacks/working-with-stacks.md:322]: Thanks for making these changes

**Comment by jantley-ibm** [comment]
Closing this PR because changes were made to the wrong file.

**Pull request closed without merging**